Sources/Testing/ABI/v0/Encoded/ABIv0.EncodedIssue.swift

```diff
@@ -22,9 +22,13 @@ extension ABIv0 {
     /// The location in source where this issue occurred, if available.
     var sourceLocation: SourceLocation?
 
+    /// The backtrace where this issue occurred, if available.
+    var _backtrace: [Backtrace.Address]?
```

Comment:
This is not formally part of the JSON schema (hence the underscore) but I realized it was missing and should be present here.